apps/complaints/fields/index.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 
 module.exports = {
-	'name-text': {
-		mixin: 'input-text'
-	}
+	'who-radio': {
```

Comment:
I know we have been using the {name}-{mixin-type} format for this, but this ties it too closely together with the mixin type, eg if we want to change a text input to a number input, or textarea, this is another place we would need to change it.